Npm package broken

The NPM package seems to be broken :

```
npm install hideShowPassword
```
```
fetch failed https://registry.npmjs.org/hideShowPassword/-/hideShowPassword-2.0.4.tgz
npm ERR! Error: 404 Not Found
```

By the way, thanks for the good work. It works very well.

--- a/README.md
+++ b/README.md
@@ -319,6 +319,7 @@ $('#my-modal').on('shown.bs.modal', function (event) {
 
 ## History
 
+* **2.0.5**: Fixing `npm install` issue ([#28](https://github.com/cloudfour/hideShowPassword/issues/28))
 * **2.0.4**: Namespaced events ([#20](https://github.com/cloudfour/hideShowPassword/issues/20)), [npm](https://www.npmjs.com/) support ([#21](https://github.com/cloudfour/hideShowPassword/issues/21))
 * **2.0.3**: Removed errant `console.log` call ([#13](https://github.com/cloudfour/hideShowPassword/issues/13))
 * **2.0.2**: `className` option now instantiates on `init` ([#11](https://github.com/cloudfour/hideShowPassword/issues/11))

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "hideShowPassword",
-  "version": "2.0.4",
+  "version": "2.0.5",
   "description": "Easily reveal or hide password field contents via JavaScript or a nifty inner toggle button. Supports touch quite nicely!",
   "main": "hideShowPassword.js",
   "scripts": {
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/cloudfour/hideShowPassword.git"
+    "url": "https://github.com/cloudfour/hideShowPassword.git"
   },
   "keywords": [
     "form",

